app/bedingungen/bedingungen.js

```diff
@@ -1,8 +1,8 @@
-import Article from "../components/Article.js";
-import Layout from "../components/Layout.js";
+"use client";
```

Comment:
Can we move client components further down the tree, where we really need them?

Reply:
all clients component are in components folder now